services/address_book_service.proto

```diff
@@ -29,37 +27,127 @@ import "transaction_response.proto";
 import "transaction.proto";
 
 /**
- * Transactions for the AddressBook Service, those HAPI APIs facilitate changes to the nodes used across the Hedera network.
- * All those transactions needs to be signed by the Hedera Council. Steps needed below:
- * 1. The node operator creates and signs the transaction with their key (the key on the node operator account)
- * 2. The node operator hands this transaction to Alex, who then gives it to the council to sign
- * 3. When signed and submitted, the server will verify that account 2 keys have signed, and the keys on the operator account have signed.
- * Hedera council should have ability to make all edits in addition to add/remove nodes
+ * The Address Book service provides the ability for Hedera network node
+ * administrators to add and remove consensus nodes. The addition or removal
```

Comment:
administrators to add, update and remove consensus nodes. The addition, updating or removal